wolfcrypt/src/sha.c

```diff
@@ -822,7 +877,7 @@ int wc_ShaFinal(wc_Sha* sha, byte* hash)
 
     XMEMCPY(hash, (byte *)&sha->digest[0], WC_SHA_DIGEST_SIZE);
 
-    (void)InitSha(sha); /* reset state */
+    ret = InitSha(sha); /* reset state */
```

Comment:
There is some history here about why we ignore the error code here, although I don't think it may mater anymore. @ejohnstown do you recall why? Is it because originally there wasn't a return code on the SHA-1 final?

Reply:
Even if the return value is ignored, I'll leave this returning a value for consistency unless otherwise advised.